chpl/chpl-resources/src/main/resources-production/jobs.xml

```diff
@@ -12,11 +12,11 @@
     </pre-processing-commands>
     <schedule>
         <!-- Delete from here to "DELETE END MARKER" after PROD push -->
-        <!--  Updates all g1g2 "ACI" MACRA measures to "PI" -->
+        <!--  Updates all MACRA measures to based on new requirements -->
         <job>
             <name>updateMacraMeasuresJob</name>
             <group>systemJobs</group>
-            <description>Updates all g1g2 ACI MACRA measures to PI</description>
+            <description>Updates all MACRA measures to based on new requirements</description>
```

Comment:
If the previous job goes to PROD before this one does we'll need to make sure that updating a job description in the jobs.xml for PROD works correctly